collector/docker/kindling-collector-config.yml

```diff
@@ -110,6 +112,8 @@ exporters:
       # send trace data in the format of ResourceSpan
       need_trace_as_span: false
     metric_aggregation_map:
+      kindling_pagefault_major_total: gauge
+      kindling_pagefault_minor_total: gauge
```

Comment:
In Prometheus, `gauge` is used to record non monotonic data, corresponding to `gaugeObserver` in OTEL. (check this in ins.recordLastValue()). Obviously, the page fault metric is monotonically increasing and cumulative pre-aggregated metric, which corresponds to `counterObserver` in OTEL. Can you add some support for this kind of data?

Reply:
It's tough work for him to add this support. We better leave this behind and might do it later. Prometheus supports "increase()" or "rate()" on this kind of metric that is `gauge` but monotonically increasing, so I think `gauge` is ok for this scenario.

Reply:
I will open a new issue for this later.